lang/python/python-mako/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
Omit or assign to yourself.
See: https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

Reply:
ok, will remove